src/bundle/Resources/public/css/alloyeditor/alloyeditor-ez.css

```diff
@@ -1044,3 +1044,7 @@
   font-size: 14px;
   line-height: 1;
 }
+
+.alloy-editor-visible {
```

Comment:
I don't know this class. Where it's added?

Reply:
This class is added for modals with `custom tag` and `anchor`.

Reply:
Thanks to this change, the `custom tag`:
- visibility of `custom tag` will have a bigger priority than buttons bar
- if the `custom tag` element exceeds the height limit, overflow appears

Reply:
This problem probably will be for other toolbars as well. What do you think to add this to the parent `ae-toolbars` and kill bug once for all?

Reply:
Adding `z-index: 1;` to `.ae-toolbars` won't solve the problem.
Child needs this attribute.
![Screenshot 2019-03-14 at 10 04 19](https://user-images.githubusercontent.com/22669715/54344117-9dc07d80-4640-11e9-8a30-4ed45268067d.png)
![Screenshot 2019-03-14 at 10 04 09](https://user-images.githubusercontent.com/22669715/54344169-a3b65e80-4640-11e9-93f2-0a17e029187f.png)